src/Abp/Text/FormattedStringValueExtracter.cs

```diff
@@ -85,6 +82,24 @@ public ExtractionResult Extract(string str, string format, bool ignoreCase = fal
             return result;
         }
 
+        private List<FormatStringToken> TokenizeFormat(string originalFormat, char splitformatCharacter)
+        {
+            if (splitformatCharacter == default(char))
+            {
+                return new FormatStringTokenizer().Tokenize(originalFormat);
+            }
+
+            var result = new List<FormatStringToken>();
+            var formats = originalFormat.Split('/');
```

Comment:
Why not used splitformatCharacter instead of '/'..?

Reply:
should be fixed with https://github.com/aspnetboilerplate/aspnetboilerplate/pull/3255/commits/ba2be843967d706cdc250c3143504d40dc0e7338